src/App.js

```diff
@@ -9,15 +9,37 @@ import Register from './pages/register';
 import Theme from './components/theme'
 import { Text } from "@chakra-ui/react"
 import useLocalStorageState from "use-local-storage-state";
+import MyGames from './pages/myGamesPage';
+import axios from 'axios';
 
 function App() {
-    const [token, setToken] = useLocalStorageState("teammateToken", null)
-    const [username, setUsername] = useLocalStorageState("teammateUsername", null)
+  const [token, setToken] = useLocalStorageState("teammateToken", null)
+  const [username, setUsername] = useLocalStorageState("teammateUsername", null)
+  const [listType, setListType] = useState(null);
+//   above is the listType to be used with GamesList component (which is rendered from OpenGamesList and MyGames components)
+const [allGamesList, setAllGamesList] = useState([])
+
+
+
+  const setAuth=(username, token)=>{
+  setToken(token)
+  setUsername(username)
+  }
+
+  useEffect(() => {
+    // setListType("allOpen")
+    axios
+        .get("https://teammate-app.herokuapp.com/session", {
+            headers: {
+                Authorization: `Token ${token}`,
+            },
+        })
+        .then((res) => {
+            console.log(res.data);
+            setAllGamesList(res.data);
+        });
+}, [token, setAllGamesList, setListType]);
```

Comment:
I don't know enough to know for sure, but would the useEffect setting the gameslist and updating the games list cause an infinite loop?

Reply:
It isn't currently causing an infinite loop